lib/emailer.ts

```diff
@@ -85,52 +85,92 @@ export async function send(story: Story): Promise<string | null> {
   return null
 }
 
-function createMessage({ postal, id, title, content }: Story) {
-  const ontario = isOntario(postal)
-  const to = CUSTOM_EMAILS || (ontario ? ONTARIO_EMAILS : CANADA_EMAILS)
-  const subject = `A COVID-19 story${ontario ? ' from Ontario' : ''}`
-  const addressee = ontario ? ONTARIO_ADDRESSEE : CANADA_ADDRESSEE
-  const jurisdiction = ontario ? 'Ontario' : 'Canada'
-  const url = `${process.env.BASE_URL}/story/${id}`
-  const params = { addressee, postal, jurisdiction, url, title, content }
```

Comment:
This logic got too complicated to do inline with nested ternaries, so I moved it into `computeDetails()`, where I can use early returns to keep things cleaner.